CMakeLists.txt

```diff
@@ -87,6 +87,11 @@ endif ()
 project(gdal LANGUAGES C CXX)
 include(CTest)
 
+# Include defaults
+if (EXISTS "${PROJECT_SOURCE_DIR}/ConfigUser.cmake")
+  include(ConfigUser.cmake)
```

Comment:
Apart from using different paths: there is an `OPTIONAL` keyword for `include` so the `if` is not needed.